packages/react-core/src/demos/ComposableMenu.md

```diff
@@ -1321,3 +1321,109 @@ MenuContextSelector = () => {
   return <Popper trigger={toggle} popper={menu} isVisible={isOpen} />;
 };
 ```
+
+### Options menu using menu components
+
+```js
+import React from 'react';
+import {
+  MenuToggle,
+  Menu,
+  MenuContent,
+  MenuList,
+  MenuItem,
+  MenuGroup,
+  Popper,
+  Tooltip,
+  Divider
+} from '@patternfly/react-core';
+import { Link } from '@reach/router';
+import SearchIcon from '@patternfly/react-icons/dist/esm/icons/search-icon';
+import pfIcon from './examples/pf-logo-small.svg';
+
+MenuOptionsMenu = () => {
+
+  const [isOpen, setIsOpen] = React.useState(false);
+  const [selected, setSelected] = React.useState(undefined);
+  const toggleRef = React.useRef();
+  const menuRef = React.useRef();
+
+  const handleMenuKeys = event => {
+    if (!isOpen) {
+      return;
+    }
+    if (menuFooterBtnRef.current.contains(event.target)) {
+      if (event.key === 'Tab') {
+        if (event.shiftKey) {
+          return;
+        }
+        setIsOpen(!isOpen);
+        toggleRef.current.focus();
+      }
+    }
+    if (menuRef.current.contains(event.target)) {
+      if (event.key === 'Escape') {
+        setIsOpen(!isOpen);
+        toggleRef.current.focus();
+      }
+    }
```

Comment:
This isn't working currently, though it looks like removing the invalid `menuFooterBtnRef` mentioned above resolves it.